scripts/paycontract.py

```diff
@@ -0,0 +1,33 @@
+def pay(pay, balance1, balance2):
+    balance1 = int(balance1)
+    balance2 = int(balance2)
+    pay = int(pay)
+    if(pay > balance1):
+        pay = 0
+    balance1 = balance1 - pay
+    balance2 = balance2 + pay
+    return [balance1, balance2]
+
+def accrueInterest(rate, balance1):
+    balance1 = int(balance1)
+    rate = float(rate)
+    print("Balance 1:", balance1)
+    print("Rate:", rate*100, "%")
+    balance1 = int(balance1 * (2.71828**rate))
+    print("Balance 1:", balance1)
+    return [balance1]
```

Comment:
Would write it like this. Could add a "Balance 1 after rate:" print after the conversion has been applied.
```suggestion
def accrueInterest(rate: float, balance1: int) -> list[int]:

    rate, balance1 = float(rate), int(balance1)

    print(f"Balance 1: {balance1}")
    print(f"Rate: {rate*100}%")

    balance1 = int(balance1 * (2.71828**rate))
    print(f"Balance 1: {balance1}")

    return [balance1]
```